plugins/packages/smtp/lib/index.ts

```diff
@@ -0,0 +1,100 @@
+import { QueryError, QueryResult, QueryService, ConnectionTestResult } from '@tooljet-plugins/common';
+import { SourceOptions, QueryOptions } from './types';
+import nodemailer from 'nodemailer';
+
+export default class Smtp implements QueryService {
+  async run(sourceOptions: SourceOptions, queryOptions: QueryOptions, dataSourceId: string): Promise<QueryResult> {
+    const nodemailerTransport = await this.getConnection(sourceOptions);
+
+    const from = queryOptions.from;
+    const from_name = queryOptions.from_name;
+    const to = queryOptions.to;
+    const subject = queryOptions.subject;
+    const textContent = queryOptions.textContent;
+    const htmlContent = queryOptions.htmlContent;
```

Comment:
try
`const { from, to, from_name, subject, textContent, htmlContent } = queryOptions`